pkg/chart/chart.go

```diff
@@ -30,12 +44,12 @@ func ReadChartConfig(basePath string, chartName string) ([]byte, error) {
 // Read reads a README YAML file and returns the Content to render.
 func ReadChartMetadata(basePath string, chartName string) (Metadata, error) {
```

Comment:
Is this reading or generating? the error is updated, but not the function name

Reply:
the chart configuration is generated using schemadocs binary but the metadata is read from the repo